bookshop/test/requests.http

```diff
@@ -62,6 +62,22 @@ Authorization: Basic alice:
   "currency": { "code": "USD" }
 }
 
+### ------------------------------------------------------------------------
+# Create book with invalid author ID -> will fail
+POST {{server}}/admin/Books
```

Comment:
Maybe we could also add an example of where a `ON DELETE CASCADE` deletes leftover children?